io/plugins/eu.esdihumboldt.hale.io.shp/src/eu/esdihumboldt/hale/io/shp/writer/ShapefileInstanceWriter.java

```diff
@@ -373,7 +374,7 @@ private String truncatePropertyName(String propName) {
 	private List<GeometryProperty<?>> traverseInstanceForGeometries(Instance instance) {
 		// find geometries in the schema.
 		InstanceTraverser traverser = new DepthFirstInstanceTraverser(true);
-		GeometryFinder geoFind = new GeometryFinder(null);
+		GeometryFinder geoFind = new GeometryFinder(getTargetCRS());
```

Comment:
Looking at the [code](https://github.com/halestudio/hale/blob/01a8d220645900b77cd7a1249aa59ab6a0273984/common/plugins/eu.esdihumboldt.hale.common.instance/src/eu/esdihumboldt/hale/common/instance/geometry/GeometryFinder.java#L83) this will assign the CRS to a geometry encountered w/o CRS.
A CRS assignment should not be done by the writer, but when the data is read. The target CRS is intended to convert the data to, but if the data has not CRS, it can't be converted.

Reply:
> Than could be possible that the project that we are testing doesn't have a CRS?

Looks like it - I guess this is not contained in a unit test but something you test manually in the UI?
You can check the source data by example in the source data view and can see there also if for a geometry CRS information is included or not.

Reply:
Also, only if there is a CRS instances can appear on the map view.

Reply:
I'm testing it from the UI. In the source data I cannot see any information about CRS. The project attached to the issue is pretty big, maybe is hidden somewhere and I cannot see it.

All the other tests related to shp are passing successfully.

Reply:
If you can share your test project somewhere (e.g. Box) I could have a look.

Reply:
Provided feedback in the issue

Reply:
I have keep the missingGeomsForSchemas list just to add into the reporter the warning that the geometry is missing. 
We can also decide to remove it and just leave the warning in the UI that informs the user that "...Feature types with no geometry will not be exported".